vantage6/node/__init__.py

```diff
@@ -430,8 +438,10 @@ def authenticate(self):
         """
         api_key = self.config.get("api_key")
 
-        keep_trying = True
-        while keep_trying:
+        success = False
+        i = 0
+        while i < 10:
+            i = i + 1
```

Comment:
So we are now trying to authenticate 10 times, then exiting. Why this change in this way?
If authentication fails because API-key is wrong, I think it should exit immediately with that message.
If authentication fails because server is offline, this procedure is quite nice, but it may be included in the error message.
Maybe we should further improve these messages in a separate issue...

Reply:
Yes that is a good point, this is a rushed fix (It was really annoying in the log files that some nodes kept trying to authenticate with a wrong API-key). But indeed, it should only re-attempt to authenticate when an connection error is raised (and maybe some http-codes like 500). I created a separate issue for this: https://github.com/IKNL/vantage6-node/issues/64